three.c_1.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
```

Comment:
Nice job, overall, but just a few things: 
- include comments!!
- you didn't handle input well